tdex/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: "3.8"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_TDEX_CONNECT_IP
+      APP_PORT: $APP_TDEX_CONNECT_PORT
+  
+  tdexd:
+    container_name: "tdexd"
+    image: docker pull ghcr.io/tdex-network/tdexd:v0.8.15@sha256:e79aff49e5220d55df7c94d01e404e11b7e5cf094b459b3e6fbb918a7f2a40d6
+    tty: true
+    restart: on-failure
+    stop_grace_period: 1m
+    ports:
+      # Replace <port> with the port that your app's web server
+      # is listening inside the Docker container. If you need to
+      # expose more ports, add them below.
+      - 9090:9090
+    volumes:
+      # Uncomment to mount your data directories inside
+      # the Docker container for storing persistent data
+      - ${APP_DATA_DIR}/tdex-data:/home/tdex/.tdex-daemon
+
+    environment:
+      # Pass any environment variables to your app for configuration in the form:
+      TDEX_LOG_LEVEL: 5
+      TDEX_EXPLORER_ENDPOINT: https://liquid.network/liquid/api
+      TDEX_OPERATOR_LISTENING_PORT: 9090
+      TDEX_TRADE_LISTENING_PORT:  9090
+      TDEX_NO_OPERATOR_TLS: "true"
+      TDEX_CONNECT_ADDR: "$DEVICE_HOSTNAME:9090"
+      TDEX_CONNECT_PROTO: http
+      #
+      # Here are all the Umbrel provided variables that you can pass through to
+      # your app
+      # System level environment variables
+      # $DEVICE_HOSTNAME - Umbrel server device hostname (e.g. "umbrel")
+      # $DEVICE_DOMAIN_NAME - A .local domain name for the Umbrel server (e.g. "umbrel.local")
+      #
+      # Tor proxy environment variables
+      # $TOR_PROXY_IP - Local IP of Tor proxy
+      # $TOR_PROXY_PORT - Port of Tor proxy
+      #
+      # App specific environment variables
+      # $APP_HIDDEN_SERVICE - The address of the Tor hidden service your app will be exposed at
+      # $APP_PASSWORD - Unique plain text password that can be used for authentication in your app, shown to the user in the Umbrel UI
+      # $APP_SEED - Unique 256 bit long hex string (128 bits of entropy) deterministically derived from user's Umbrel seed and your app's ID
+  dashboard:
+    container_name: dashboard
+    image: ghcr.io/tdex-network/dashboard:latest
+    restart: unless-stopped
```

Comment:
We usually default to `on-failure` so that the container only restarts with a non-zero status code (which indicates an error).